packages/prosemirror-lwdita-demo/src/demo-plugin.ts

```diff
@@ -121,16 +121,16 @@ export function openFileMenuItem(localization: Localization): MenuElement {
  * @returns The menu element for publishing the file.
  */
 export function publishFileMenuItem(config: Config, localization: Localization, urlParams: URLParams): MenuElement {
-  const storedFileName = localStorage.getItem('fileName') ? localStorage.getItem('fileName') : 'Petal';
+  const storedFileName = urlParams? urlParams.source.split('/').slice(-1) : 'Petal.xml';
```

Comment:
This is setting the default filename for all uploaded files when Petal is used as a stand-alone editor without the GItHub integration. The returned type will be an array instead of a string, but we need to display a string here.